### ISSUE TITLE
Results

### ISSUE THREAD

**Issue by phellonchen** [ShannonAI/OpenViDial#25]
Hi,
I run your code (FV). I get the gen.out file. The result seems that (Last line in gen.out):
Generate test with beam=5: BLEU4 = 0.40, 6.2/0.5/0.1/0.1 (BP=1.000, ratio=2.087, syslen=810940, reflen=388568)
Diversity-1: 0.0006535270732838133
Diversity-2: 0.0031576051690483616
Diversity-3: 0.006970750791777929
Diversity-4: 0.010780433006226303
StopWords%: 0.3479316484665542; StopWords/Sent: 5.507739454627082
AvgLength: 15.829946711951749
Are the results correct?

**Comment by YuxianMeng** [comment]
@phellonchen Hi, we found the current cnn/rcnn features for valid/test set is not correct, we will update it soon!

**Comment by YuxianMeng** [comment]
@phellonchen We updated cnn/rcnn feature of valid/test dataset today! We achieved 0.95/1.14/1.18  BLEU4 for NV/CV/FV, respectively. Other stats are coming soon.

**Comment by wjczf123** [comment]
@YuxianMeng I seems that I also need use new train.objects.mmap.splitao.mmap? And there is no space left on my device. 
Can I still run normally after I delete origin_dir/train_images?

**Comment by YuxianMeng** [comment]
@wjczf123 Yes, you're right. Only data in preprocessed_dir are used during training and inference.

**Comment by wjczf123** [comment]
@YuxianMeng OK. Thank you a lot.
